pandas/util/_test_decorators.py

```diff
@@ -140,6 +140,17 @@ def skip_if_no(
     )
 
 
+def skip_if_installed(
```

Comment:
are u using this?

Reply:
https://github.com/pandas-dev/pandas/pull/26816/files#diff-56c44e2fe6ac59a333e18f097db07524R57.

We should probably have a basic test like this for all our optional deps. Thoughts?

It can't go in `test_pytables.py` since that does an `importorskip` at the top of the file.

Reply:
Alternatively, we could *try* monkeypatching / mocking the module, but I've never had luck working around Python's import caching.

Reply:
not sure i understand? we already test all of the optional depa but using them
are you suggesting something else?

Reply:
> are you suggesting something else?

Yeah, testing when that *aren't* present.